save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/HttpUtils.kt

```diff
@@ -21,6 +21,12 @@ private const val DEFAULT_BUFFER_SIZE = 4096
 private const val PORT_RANGE_FROM = 20000
 private const val PORT_RANGE_TO = 30000
 
+/**
+ * Subdomain of all save-demo-agents
+ */
+const val DEMO_SUBDOMAIN_NAME = "demo-agent-service"
```

Comment:
You could actually avoid duplication of this string like this: You could place it into `values.yaml`, reference in YAML declaration for Service and pass to `save-demo` either via ConfigMap or env var.

Reply:
Done